### ISSUE TITLE
Neovim seems to be working too well

### ISSUE THREAD

**Issue by dpacbach** [neovim/neovim#20237]
### Neovim version (nvim -v)

0.7.2

### Vim (not Nvim) behaves the same?

no

### Operating system/version

Linux

### Terminal name/version

alacritty

### $TERM environment variable

xterm-256color

### Installation

Custom build

### How to reproduce the issue

To reproduce the issue, build neovim, then port your vimrc to Lua, and observe how life magically becomes better.

### Expected behavior

Unexpected problems, frustration, crashes, etc.

### Actual behavior

Bliss.

**Comment by dpacbach** [comment]
Today I got fed up with vimscipt (which is saying a lot since I've been writing/avoiding it for 10 years) and I just ported all of my scripts and PDE setup today to neovim/lua in under a couple of hours, which was not easy, but desperation breeds genius.  As of now, all of my PDE environment/setup is now rewritten in neovim Lua and I'm actually looking forward to further customizing my editor.  You guys are doing the lord's work.  Keep it up.
